modules/tinymce/src/themes/silver/main/ts/ui/core/complex/BespokeNumberInput.ts

```diff
@@ -37,9 +38,14 @@ const createBespokeNumberInput = (editor: Editor, backstage: UiFactoryBackstage,
 
   const changeValue = (f: (v: number, step: number) => number, fromInput: boolean, focusBack: boolean): void => {
     const text = getValueFromCurrentComp(currentComp);
-    const parsedText = Dimension.parse(text, [ 'unsupportedLength' ]);
+    const parsedText = Dimension.parse(text, [ 'unsupportedLength', 'empty' ]);
     const value = parsedText.map((res) => res.value).getOr(0);
-    const unit = parsedText.map((res) => res.unit).getOr('');
+    const defaultUnit = getFontSizeInputDefaultUnit(editor);
+    const unit = parsedText.map((res) => res.unit).fold(
```

Comment:
This could be one with `parsedText.getOr(defaultUnit)`.

Reply:
Ahh nevermind I misread that. Yes it needs to be a fold or a map and a getOr.

Reply:
@spocke I refactored it with a `filter`, now it should be more readable, have a check